healthcare_professional_entity.go

```
@@ -0,0 +1,515 @@
+package yext
+
+// TODO
+// * Need better custom field accessors and helpers
+// * The API will accept some things and return them in a different format - this makes diff'ing difficult:
+// ** Phone: Send in 540-444-4444, get back 5404444444
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+const ENTITYTYPE_HEALTHCAREPROFESSIONAL EntityType = "healthcareProfessional"
+
+const ENTITYTYPE_HEALTHCAREPROFESSIONAL_CF EntityType = "HEALTHCARE_PROFESSIONAL"
```

What does CF stand for?

---

Basically if we don’t expect a panic normally it might be better to just let it bubble

---

CF stands for Custom Field, we have to support two separate API casings right now because of the custom fields create endpoint. There's an item in Apollo's backlog to fix this.